src/components/modules/AverageTimeReports/AverageTimeReports.tsx

```diff
@@ -0,0 +1,104 @@
+/*eslint-disable*/
+import moment from 'moment';
+import React, { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import styled from 'styled-components';
```

Comment:
Use Tailwind instead of `styled-components` for styling. I would like to keep everything consistent and we already use Tailwind in the app.